packages/components/src/select-control/types.ts

```diff
@@ -22,9 +22,12 @@ type SelectControlBaseProps = Pick<
 	| 'suffix'
 > &
 	Pick< BaseControlProps, 'help' | '__nextHasNoMarginBottom' > & {
-		onBlur?: ( event: FocusEvent< HTMLSelectElement > ) => void;
-		onFocus?: ( event: FocusEvent< HTMLSelectElement > ) => void;
```

Comment:
Removing these explicit event listener props since they are mixed in here:

https://github.com/WordPress/gutenberg/blob/2c01c7dd9dbea28e8afb85078ac5aef670f206cc/packages/components/src/select-control/index.tsx#L45-L47

This will remove them from the props table and auto-generated code snippets in Storybook, which is good because we have no reason to explicitly show `onBlur` and `onFocus`.